### PR TITLE
Update account-lockout-threshold.md

### LINKED ISSUE #7018: Exclusion of built in administrator
It would be helpful if this page noted that a Built in Domain Administrator is exempted from these policies. Although this is documented on this older page:

https://techcommunity.microsoft.com/t5/core-infrastructure-and-security/built-in-administrator-account-lockout/ba-p/259077

I have regularly noted community members who are unaware of this fact, and will reference this page as some form of indication that the Built In Administrator in fact should be impacted, as the wording implies it applies to all accounts.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 3974da73-1577-dbaf-00ec-f6fb6eee35df
* Version Independent ID: a562ec45-0c96-733b-2bef-c12f627dc02f
* Content: [Account Lockout Policy (Windows 10) - Windows security](https://docs.microsoft.com/en-us/windows/security/threat-protection/security-policy-settings/account-lockout-policy)
* Content Source: [windows/security/threat-protection/security-policy-settings/account-lockout-policy.md](https://github.com/MicrosoftDocs/windows-itpro-docs/blob/public/windows/security/threat-protection/security-policy-settings/account-lockout-policy.md)
* Product: **w10**
* Technology: **windows**
* GitHub Login: @Dansimp
* Microsoft Alias: **dansimp**

### PR DESCRIPTION
**Pull request by lindspea**
https://github.com/MicrosoftDocs/windows-itpro-docs/issues/7018
Added note.
Closes #7018

### CODE REVIEW

**Comment by illfated** [comment]
Just in case you forgot, you can change the state of the related issue ticket to be closed automatically, by adding the following to your first comment in this PR (on a separate line):

Closes #7018

**Comment by e0i** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Review by illfated — APPROVED** [review]
Looks good to me.

File: windows/security/threat-protection/security-policy-settings/account-lockout-threshold.md [hunk 1]
```diff
@@ -87,6 +87,9 @@ For more information about Windows security baseline recommendations for account
 
 This section describes how an attacker might exploit a feature or its configuration, how to implement the countermeasure, and the possible negative consequences of countermeasure implementation.
 
+> [!NOTE]
+> A lockout threshold policy will apply to both local member computer users and Domain Users, in order to allow mitigation of issues as described under "Vulnerability". The Built-In Administrator account however, whilst a highly privileged account, has a different risk profile and is excluded from this policy. This ensures there is no scenario where an administrator cannot logon to remediate an issue. As an administrator, there are additional mitigation strategies available, such as a strong password. See also [Appendix D: Securing Built-In Administrator Accounts in Active Directory](https://docs.microsoft.com/windows-server/identity/ad-ds/plan/security-best-practices/appendix-d--securing-built-in-administrator-accounts-in-active-directory).
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/security-policy-settings/account-lockout-threshold.md:91]:
```suggestion
> A lockout threshold policy will apply to both local member computer users and domain users, in order to allow mitigation of issues as described under "Vulnerability". The built-in Administrator account, however, whilst a highly privileged account, has a different risk profile and is excluded from this policy. This ensures there is no scenario where an administrator cannot sign in to remediate an issue. As an administrator, there are additional mitigation strategies available, such as a strong password. See also [Appendix D: Securing Built-In Administrator Accounts in Active Directory](https://docs.microsoft.com/windows-server/identity/ad-ds/plan/security-best-practices/appendix-d--securing-built-in-administrator-accounts-in-active-directory).
```

Comment (JohanFreelancer9) [reply windows/security/threat-protection/security-policy-settings/account-lockout-threshold.md:91]: Please change "Domain Users" to "domain users", "The Built-In Administrator account however," to "The built-in Administrator account, however,", and "logon" to "sign in".

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by nam31** [comment]
Hi @Dansimp PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**